src/core/tsi/fake_transport_security.cc

```diff
@@ -474,12 +474,21 @@ static void fake_zero_copy_grpc_protector_destroy(
   gpr_free(impl);
 }
 
+static tsi_result fake_zero_copy_grpc_protector_max_frame_size(
```

Comment:
why do we need to change this file? 

Envoy only needs to use fake handshaker (not this fake TSI).

Reply:
Envoy/ALTS has a unit test (https://cs.corp.google.com/piper///depot/google3/third_party/envoy/src/test/extensions/transport_sockets/alts/tsi_socket_test.cc?dr&q=tsi_socket_test.cc&g=0&l=27) that depends on fake TSI handshaker. Without the change in this PR, we cannot assert the return value of tsi_zero_copy_grpc_protector_max_frame_size() in [here](https://github.com/envoyproxy/envoy/pull/15297/files#diff-fa553b38a3f33ebec805224d5837609cf506dff076071518b0e26c153aa6650bR140-R142)

Reply:
Got it. Thanks for clarifying.